src/test/ui/error-trait/error-in-panic.rs

```diff
@@ -0,0 +1,50 @@
+// run-fail
```

Comment:
Can you run rustfmt on this file? (It isn't run by default on tests, because sometimes the formatting is relevant to the test.)

Reply:
I ran `./x.py fmt src/test/ui/error-trait/error-in-panic.rs` & I didn’t pick up any changes to this file. Is there something else I should do instead?